packages/app/features/top/ChannelScreen.tsx

```diff
@@ -147,7 +147,7 @@ export default function ChannelScreen({
   } = store.useChannelPosts({
     enabled: !!channel && !channel?.isPendingChannel,
     channelId: currentChannelId,
-    count: 50,
+    count: 15,
```

Comment:
cc @dnbrwstr – I saw that the previous value here was explicitly set in one of your commits – do you see any bad effect of using a smaller value here? (more context in main PR thread but gist is that loading too much alongside `maintainVisibleContentPosition` was causing white flashes (i think!)

Reply:
Nope, whatever works, works! Just trying to tune for max perf.